readme.md

```diff
@@ -24,13 +30,14 @@ Create a pages with product card using `flexbox` basing on [the mockup](https://
 --> [CHECKLIST](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
 
 ### Tips & Hints
+
 - Add **ALL** `data-qa` attributes required in the task
-- Reuse `stars` block from [Stars task](https://github.com/mate-academy/layout_stars)
-and keep it operational. If you change `stars--4` modifier to `stars--2` or
-other - number of selected stars should change correctly.
+- Reuse `stars__item` block from [stars\_\_item task](https://github.com/mate-academy/layout_stars)
+  and keep it operational. If you change `stars__item--4` modifier to `stars__item--2` or
+  other - number of selected stars\_\_item should change correctly.
 - Item with text 'Купить' should be a link.
```

Comment:
you can name you classes differently, but there is no need to change the readme file lime this